.travis.yml

```diff
@@ -7,3 +7,9 @@ services:
 script:
   - docker build -t riotdocker .
   - docker image ls riotdocker:latest
+  - git clone --depth 1 https://github.com/RIOT-OS/RIOT -b 2019.01
+  - sed -i '35i\ \ BOARDS \\' RIOT/makefiles/docker.inc.mk
+  - DOCKER_IMAGE=riotdocker:latest
+    BUILD_IN_DOCKER=1
+    BOARDS="arduino-uno esp32-wroom-32 hifive1 msb-430h native pic32-wifire samr21-xpro"
```

Comment:
Does it make sense to add also `native:llvm` and `samr21-xpro:llvm`?

Reply:
done